### DISCUSSION TITLE
Merging blocked by missing successful deployment even though deployment was successful, and GitHub knows it

### DISCUSSION THREAD

**Discussion by prplecake** [community/community#76616] [Pull Requests]
### Select Topic Area

Bug

### Body

Screenshot says it all. Successful Preview deployment reported at top, merging blocked by missing the deployment GitHub shows is successful.

<img width="728" alt="Screenshot 2023-11-22 at 07 30 39" src="https://github.com/community/community/assets/83595468/1f35105b-64de-472f-9b2b-1ebbbca50563">

Using Vercel GitHub integration - Vercel handled creating the environments for me in GitHub.

**Comment by prplecake** [comment]
See also https://github.com/orgs/vercel/discussions/1886

**Reply by dk-flpra** [reply]
This page is 404 now. Any luck on finding the fix?

**Comment by harshai07** [comment]
It seems like you're encountering an issue on GitHub related to a successful Preview deployment not reflecting in the merge status, preventing you from completing the merge. To address this bug, follow these steps:

Check Deployment Status:

Verify that the deployment GitHub shows as successful is indeed reflected in the GitHub interface.
Confirm the status of the deployment matches the reported success.
Review Repository Settings:

Ensure that your repository settings and branch protection rules are configured correctly.
Check if there are any specific requirements or conditions for merging, such as required checks or deployments.
Refresh the Page:

Sometimes, the interface might not update in real-time. Try refreshing the page to see if the correct status is reflected.
Review GitHub Actions Workflow:

If your deployment is part of a GitHub Actions workflow, review the workflow file to ensure it is correctly configured.
Check for any errors or warnings in the GitHub Actions logs.
Check GitHub Status:

Visit GitHub's status page to check if there are any ongoing incidents or outages affecting deployments.
Clear Browser Cache:

Clear your browser cache to ensure that you are viewing the most up-to-date information.
Re-deploy or Trigger Deployment:

If possible, try re-deploying or triggering a new deployment to see if it updates the deployment status.
Contact GitHub Support:

If the issue persists and none of the above steps resolve it, consider reaching out to GitHub Support for assistance. Provide them with relevant details and screenshots to help them understand the problem.
When reporting the bug to GitHub Support, be sure to include:

Detailed steps to reproduce the issue.
The exact error message or unexpected behavior you are encountering.
Any relevant screenshots or error logs.
Remember that GitHub's support team is generally responsive and can provide targeted assistance based on the specifics of your case.

**Reply by prplecake** [reply]
Get out of here with your gpt generated crap.

**Reply by queenofcorgis** [reply]
Hi @harshai07 , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by Bloke-Bloke** [reply]
change preview to lowercase

**Reply by Bhushan-WhiteBoxData** [reply]
Any luck with this @prplecake ? This seems to have been an issue for us since yesterday. We haven't made any changes at all on Github or Vercel which we use for deployment. 
We even recreated the PR multiple times to see if it would resolve but no luck so far! This is a major blocker for us. 

Also, are there any downsides to Force Merging the PR?

**Reply by reefbreak** [reply]
I'm having the same problem, i think there's a capitalisation issue. However, when trying to rename the branch it says it already exists (so ignoring capitalisation).

I'm just having to force merge at this point, any idea on how to resolve (except for set the whole thing up again using lower case branch names and rules?)

**Reply by rmcsharry** [reply]
This has started happening to us also, and now have to force merge. 

It happens every time so now I'm just turning off the checks, which is risky but we can't keep having these hold-ups, it's crazy.

**Reply by rmcsharry** [reply]
As this is only in 'discussions' how can I check if this is recorded as an official bug? Is it an official bug? Surely must be since several people have no encountered it.

But is it specific to Vercel I wonder? And therefore a bug with them and not with github.

**Reply by GunnarHolwerda** [reply]
We are having this issue as well

**Reply by 1Steamwork1** [reply]
Going to try the following. Wondering if anyone has had any success with any of these re-triggering Github to fetch the Preview Deployment status.

1. Marking the PR as Draft, Re-open it, and get it Approved.
2. Marking the PR as Closed, Re-open it, and get it Approved.
3. Playing around with the Github Webhook settings for your Vercel integration.

**Reply by NatoBoram** [reply]
I'm having the same problem. Deployment is clearly actively, I can click on it and view it.

<img width="1074" height="1494" alt="Screenshot_20251219-204555" src="https://github.com/user-attachments/assets/2905ef21-109a-4bb9-ab21-0f1d792299ed" />

**Comment by slesaad** [comment]
This is still an issue

**Comment by alex-rupprecht** [comment]
Still an issue

**Comment by HugoE-Ambrey** [comment]
We are also having this issue

**Comment by HugoE-Ambrey** [comment]
Frustrating that this is still happening:

<img width="928" height="607" alt="image" src="https://github.com/user-attachments/assets/dff939dd-65fa-4ae8-b8ab-d5fe669eeb9c" />

**Comment by LouisCullen** [comment]
We're facing this issue too 

<img width="1614" height="500" alt="CleanShot 2026-05-13 at 09 03 56@2x" src="https://github.com/user-attachments/assets/f14e0ec8-df1a-47d8-843b-c06635c7d2b9" />

**Comment by josebenitezg** [comment]
still an issue!

**Comment by rafaelcmrj** [comment]
+1

**Comment by muhzain** [comment]
+1
